dimager/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Notice how Django-generated files often start with this import?  It's a good thing to do for Python 2-3 compatibility